home_task7/EBook.js

```diff
@@ -0,0 +1,30 @@
+import Book from "./Book.js"
+
+class EBook extends Book{
+    constructor(title, author, year, format){
+        super(title, author, year)
+        this._format = format
+    }
+    get format(){
+        if(this._format !== 'number')
+        return this._format
+    }
+    set format(format){
+        if (format = "png"){
+            console.log("It`s a picture format")
+            return
+        }
+        this._format = format
+    }
+    static fromBook(book, fileFormat){
+        return new EBook(book.title, book._author, book._year, fileFormat)
```

Comment:
do not use protected properties outside a class, use getters

Reply:
done